_topic_maps/_topic_map.yml

```
@@ -989,6 +989,8 @@ Topics:
     File: cert-manager-operator-install
   - Name: Managing certificates with an ACME issuer
     File: cert-manager-operator-issuer-acme
+  - Name: Enabling monitoring for the cert-manager Operator for Red Hat OpenShift
```

```suggestion
  - Name: Enabling monitoring for cert-manager
```
As we're exposing the metrics for the operand instead of the operator, might want to name it differently to remove the operator phrase.